index.html

```diff
@@ -0,0 +1,1249 @@
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+                <div class="code-container">
+                    <h2>Grids-3</h2>
+                    <div class="grid-3">
+                        <div class="col">1</div>
+                        <div class="col">2</div>
+                        <div class="col">3</div>
+                    </div>
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%2874%2C118%2C159%2C1%29&t=seti&wt=none&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid-3%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522col%2522%253E1%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522col%2522%253E2%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522col%2522%253E3%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </section>
+
+            <!-- Grids -->
+
+            <!-- Text Utilities -->
+
+            <section id="text-utilities">
+                <div class="code-container">
+                    <h2 class="component-title">Headings</h2>
+
+                    <h1 class="h1">Heading 1</h1>
+                    <h2 class="h2">Heading 2</h2>
+                    <h3 class="h3">Heading 3</h3>
+                    <h4 class="h4">Heading 4</h4>
+                    <h5 class="h5">Heading 5</h5>
+                    <h6 class="h6">Heading 6</h6>
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%2874%2C118%2C159%2C1%29&t=seti&wt=none&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ch1%2520class%253D%2522h1%2522%253EHeading%25201%253C%252Fh1%253E%250A%253Ch2%2520class%253D%2522h2%2522%253EHeading%25202%253C%252Fh2%253E%250A%253Ch3%2520class%253D%2522h3%2522%253EHeading%25203%253C%252Fh3%253E%250A%253Ch4%2520class%253D%2522h4%2522%253EHeading%25204%253C%252Fh4%253E%250A%253Ch5%2520class%253D%2522h5%2522%253EHeading%25205%253C%252Fh5%253E%250A%253Ch6%2520class%253D%2522h6%2522%253EHeading%25206%253C%252Fh6%253E"
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+                <div class="code-container">
+                    <h2>small text</h2>
+                    <h1 class="h1">Heading 1 <small>secondary text</small></h1>
+                    <h2 class="h2">Heading 2 <small>secondary text</small></h2>

```

Comment:
This Heading has also an extra margin on the bottom.

Reply:
yeah to give some space at the margin